docs/settings.md

```diff
@@ -33,10 +33,17 @@ $ pip install uvicorn[watchgodreload]
 
 ## Logging
 
-* `--log-config <path>` - Logging configuration file.
+* `--log-config <path>` - Logging configuration file. **Options:** *`dictConfig()` formats: .json, .yaml*. Any other format will be processed with `fileConfig()`. Set the `formatters.default.use_colors` and `formatters.access.use_colors` values to override the auto-detected behavior.
+    * If you wish to use a YAML file for your logging config, you will want to specify the optional `yamllogconfig` dependency to ensure YAML support:
+
+    ```
+    $ pip install uvicorn[yamllogconfig]
+    ```
+
```

Comment:
sorry to get back on this that late :) time flies, work is insane,

we can remove this part now package install is merged